### ISSUE TITLE
Manifest zip URLs returning unzipped data

### ISSUE THREAD

**Issue by mwild1** [leafo/luarocks-site#78]
luarocks checks for a version of the manifest file with a '.zip' extension. Currently luarocks.org seems to be serving uncompressed data at these URLs for per-user manifests, which breaks luarocks.

For example:
```
~$ HEAD http://luarocks.org/manifests/mattj/manifest-5.1.zip
200 OK
Connection: close
Date: Wed, 06 Jan 2016 18:45:52 GMT
Server: nginx/1.6.3
Content-Type: text/x-lua
Client-Date: Wed, 06 Jan 2016 18:45:52 GMT
```

luarocks tries to decompress the file, but fails, leading to:

```
~$ sudo luarocks install verse --server=http://luarocks.org/manifests/mattj
Warning: Failed searching manifest: Failed extracting manifest file
```

**Comment by leafo** [comment]
should be fixed when we resolve #77, we're accidentally returning 200 for a .zip request to the server.

**Comment by leafo** [comment]
fixed now
